--- origsrc/couchdb-glib-0.6.91/couchdb-glib/Makefile.am	2010-05-24 09:58:07.000000000 -0500
+++ src/couchdb-glib-0.6.91/couchdb-glib/Makefile.am	2010-06-08 03:35:38.420621000 -0500
@@ -108,9 +109,8 @@ Couchdb-1.0.gir: libcouchdb-glib-1.0.la
         --pkg json-glib-1.0 \
         --output $@ \
         -I$(top_srcdir) \
-        $(h_DATA) \
-        $(libcouchdb_glib_1_0_la_headers) \
-        $(libcouchdb_glib_1_0_la_sources)
+        $(addprefix $(srcdir)/,$(libcouchdb_glib_1_0_la_headers)) \
+        $(addprefix $(srcdir)/,$(libcouchdb_glib_1_0_la_sources))
 
 Couchdb-1.0.typelib: Couchdb-1.0.gir
 	LD_LIBRARY_PATH=.libs$${LD_LIBRARY_PATH:+:$$LD_LIBRARY_PATH} \
--- origsrc/couchdb-glib-0.6.91/desktopcouch-glib/Makefile.am	2010-04-29 09:32:12.000000000 -0500
+++ src/couchdb-glib-0.6.91/desktopcouch-glib/Makefile.am	2010-06-08 04:12:22.335677700 -0500
@@ -47,7 +48,7 @@ Desktopcouch-1.0.gir: libdesktopcouch-gl
 	--namespace Desktopcouch --nsversion=1.0 \
 	--add-include-path=$(srcdir) \
 	--add-include-path=. \
-	--add-include-path=$(top_srcdir)/couchdb-glib \
+	--add-include-path=$(top_builddir)/couchdb-glib \
 	--include=GObject-2.0 \
 	--include=Json-1.0 \
 	--include=Couchdb-1.0 \
@@ -57,14 +58,13 @@ Desktopcouch-1.0.gir: libdesktopcouch-gl
 	--output $@ \
 	-I$(top_srcdir)/couchdb-glib \
 	-l$(top_builddir)/couchdb-glib/libcouchdb-glib-1.0.la \
-	$(h_DATA) \
-	$(libdesktopcouch_glib_1_0_la_headers) \
-	$(libdesktopcouch_glib_1_0_la_SOURCES)
+	$(addprefix $(srcdir)/,$(libdesktopcouch_glib_1_0_la_headers)) \
+	$(addprefix $(srcdir)/,$(libdesktopcouch_glib_1_0_la_SOURCES))
 
 Desktopcouch-1.0.typelib: Desktopcouch-1.0.gir
 	LD_LIBRARY_PATH=.libs$${LD_LIBRARY_PATH:+:$$LD_LIBRARY_PATH} \
 	$(INTROSPECTION_COMPILER) \
-	--includedir=$(top_srcdir)/couchdb-glib \
+	--includedir=$(top_builddir)/couchdb-glib \
 	--includedir=. \
 	$(INTROSPECTION_COMPILER_OPTS) $< -o $(@F)
 
