--- origsrc/couchdb-glib-0.6.91/couchdb-glib/Makefile.am	2010-05-24 09:58:07.000000000 -0500
+++ src/couchdb-glib-0.6.91/couchdb-glib/Makefile.am	2010-06-08 03:35:38.420621000 -0500
@@ -66,6 +66,7 @@ libcouchdb_glib_1_0_la_LIBADD =		\
 	-luuid
 
 libcouchdb_glib_1_0_la_LDFLAGS =	\
+	-no-undefined			\
 	-version-info $(LIBCOUCHDBGLIB_CURRENT):$(LIBCOUCHDBGLIB_REVISION):$(LIBCOUCHDBGLIB_AGE)
 
 hdir = $(includedir)/couchdb-glib-1.0
--- origsrc/couchdb-glib-0.6.91/desktopcouch-glib/Makefile.am	2010-04-29 09:32:12.000000000 -0500
+++ src/couchdb-glib-0.6.91/desktopcouch-glib/Makefile.am	2010-06-08 04:12:22.335677700 -0500
@@ -23,6 +23,7 @@ libdesktopcouch_glib_1_0_la_LIBADD =
 	$(DESKTOPCOUCH_GLIB_LIBS)
 
 libdesktopcouch_glib_1_0_la_LDFLAGS =	\
+	-no-undefined \
 	-version-info $(LIBCOUCHDBGLIB_CURRENT):$(LIBCOUCHDBGLIB_REVISION):$(LIBCOUCHDBGLIB_AGE)
 
 hdir = $(includedir)/desktopcouch-glib-1.0
